src/GitHub.VisualStudio/Views/Dialog/ForkRepositoryExecuteView.xaml

```diff
@@ -89,7 +89,7 @@
                                Visibility="{Binding Error, Converter={ui:NullToVisibilityConverter}}" 
                                HorizontalAlignment="Left" />
                     <StackPanel Orientation="Horizontal" HorizontalAlignment="Right">
-                      <Button HorizontalAlignment="Right" Margin="0 0 4 0">Back</Button>
+                      <Button HorizontalAlignment="Right" Margin="0 0 4 0" Click="backButton_OnClick">Back</Button>
```

Comment:
Could you use `Command="{Binding BackCommand}"` here instead of `Click="backButton_OnClick"`?

Reply:
I tried because you mentioned it, but it causes a crash. I'm not sure why.

Reply:
This should definitely work.